src/store/macros/sagas.js

```diff
@@ -180,16 +180,6 @@ export function* handleNodeValueUpdate (action) {
     yield put(projectError(`Failed to process macro: ${error.message}`))
   }
 }
-export function* handleNodeRangeUpdate (action) {
-  const p = action.payload
-  const node = yield select(getNode, p.id)
-  if (p.value.min) {
-    node.min = p.value.min
-  }
-  if (p.value.max) {
-    node.max = p.value.max
-  }
-}
```

Comment:
Although this code has been removed, it's worth noting that you should always dispatch an action to modify the state inside the reducer (using immutable methods) rather than directly modify the variable as you are doing here.

Reply:
I take it that that is to help with undo history or something similar?